src/main/java/org/commcare/suite/model/Action.java

```diff
@@ -30,6 +30,7 @@
     private XPathExpression relevantExpr;
     private String iconReferenceForActionBar;
     private boolean isAutoLaunching;
+    private boolean redoLast;
```

Comment:
Does this need a getter and setter? I'm getting this locally:
```
> Task :compileJava
warning: unknown enum constant When.MAYBE
  reason: class file for javax.annotation.meta.When not found
/Users/jschweers/Documents/formplayer/src/main/java/org/commcare/formplayer/beans/menus/EntityListResponse.java:299: error: invalid method reference
        return formatActionsWithFilter(session, Action::isRedoLast);
```

Reply:
Ahh weird, can you make sure you are on Java 8 ? It's compiling successfully for me. The error seems to happen because of the method reference syntax `Action::isRedoLast` but also weird that you are not getting the same for `Action::isAutoLaunching`.

Reply:
I worked around it with a local modification for a couple of days and then the error just disappeared, so... 🤷  ? At least it's good now.